locksmithctl/locksmithctl.go

```diff
@@ -215,6 +219,10 @@ func getClient() (*lock.EtcdLockClient, error) {
 		Endpoints: globalFlags.Endpoints,
 		Transport: transport,
 	}
+	if globalFlags.EtcdUsername != "" && globalFlags.EtcdPassword != "" {
```

Comment:
good point, looking at https://github.com/coreos/locksmith/blob/master/Godeps/_workspace/src/github.com/coreos/etcd/client/client.go#L181-L186 it's not necessary.